app/internal/notify/client.go

```diff
@@ -77,54 +84,90 @@ type errorItem struct {
 func (c *Client) TemplateID(name string) string {
 	if c.isProduction {
 		switch name {
-		case "MLPA Beta signature code":
+		case "MLPA Beta signature code - Email":
 			return "95f7b0a2-1c3a-4ad9-818b-b358c549c88b"
+		case "MLPA Beta signature code - SMS":
+			return "a0997cbf-cfd9-4f01-acb2-f33b07074662"
 		}
 	} else {
 		switch name {
-		case "MLPA Beta signature code":
+		case "MLPA Beta signature code - Email":
 			return "7e8564a0-2635-4f61-9155-0166ddbe5607"
+		case "MLPA Beta signature code - SMS":
+			return "0aa5b61c-ef30-410a-8473-915df9d343a5"
 		}
 	}
 
 	return ""
 }
 
 func (c *Client) Email(ctx context.Context, email Email) (string, error) {
-	token, err := jwt.NewWithClaims(jwt.SigningMethodHS256, &jwt.RegisteredClaims{
-		Issuer:   c.issuer,
-		IssuedAt: jwt.NewNumericDate(c.now()),
-	}).SignedString(c.secretKey)
+	var buf bytes.Buffer
+	if err := json.NewEncoder(&buf).Encode(email); err != nil {
+		return "", err
+	}
+
+	req, err := c.request(ctx, "/v2/notifications/email", buf)
+
+	resp, err := c.doer.Do(req)
 	if err != nil {
 		return "", err
 	}
+	defer resp.Body.Close()
 
-	var buf bytes.Buffer
-	if err := json.NewEncoder(&buf).Encode(email); err != nil {
+	var r response
+	if err := json.NewDecoder(resp.Body).Decode(&r); err != nil {
 		return "", err
 	}
 
-	req, err := http.NewRequestWithContext(ctx, http.MethodPost, c.baseURL+"/v2/notifications/email", &buf)
-	if err != nil {
+	if len(r.Errors) > 0 {
+		return "", r.Errors
+	}
+
+	return r.ID, nil
+}
+
+func (c *Client) Sms(ctx context.Context, sms Sms) (string, error) {
+	var buf bytes.Buffer
+	if err := json.NewEncoder(&buf).Encode(sms); err != nil {
 		return "", err
 	}
-	req.Header.Add("Content-Type", "application/json")
-	req.Header.Add("Authorization", "Bearer "+token)
+
+	req, err := c.request(ctx, "/v2/notifications/sms", buf)
```

Comment:
you could probably move all the common code below into this, and call it something like `doRequest`